Sources/Mantis/CropView/CropView.swift

```diff
@@ -29,6 +29,7 @@ protocol CropViewDelegate: AnyObject {
     func cropViewDidBecomeUnResettable(_ cropView: CropViewProtocol)
     func cropViewDidBeginResize(_ cropView: CropViewProtocol)
     func cropViewDidEndResize(_ cropView: CropViewProtocol)
+    func cropViewDidBeginCrop(cropView: CropViewProtocol)
```

Comment:
Please use `func cropViewDidBeginCrop(_ cropView: CropViewProtocol)` to keep consistent with other function definitions in the Protocol.